src/Http/SessionAccessToken.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Http;
+
+class SessionAccessToken extends AccessToken
+{
+    public static $type = 'session';
+
+    protected static $lifetime = 3600;
```

Comment:
```suggestion
    protected static $lifetime = 60 * 60;  // 1 hour
```

In constency with `RememberAccessToken`